.travis.yml

```diff
@@ -1,4 +1,25 @@
 language: ruby
+sudo: false
+
+cache: bundler
+
+bundler_args: --jobs 3 --retry 3 --binstubs
+
+before_install:
+  - "echo 'gem: --no-ri --no-rdoc' > ~/.gemrc"
+  - gem uninstall -aIx bundler
+  - gem install bundler -v 1.13.1
+
+install:
+- bundle install --path vendor/bundle
+
+script: bundle exec rake
+
 rvm:
-  - 2.0.0
-  - 2.1.0
+- 2.2.2
+- 2.2.3
+- 2.3.1
```

Comment:
Also, can those be specified as major.minor so that we get the latest patch automatically? I can't recall if Travis allows this.